declarations/Bardeen.history.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Terms of Service": [
+    {
+      "fetch": "https://www.bardeen.ai/terms-of-service",
+      "select": [
+        ".content-text"
+      ],
+      "validUntil": "2024-07-03T06:30:00Z"
```

Comment:
As for the privacy policy correction PR, I can't find the same `validUntil`, i would go for `2024-07-03T06:30:04Z`

Reply:
My copy-pasting mangled the last digits for some reason and I just thought the seconds don't really matter… If you think it's important that it matches, okay to make it match!

Reply:
The only thing that worries me about this value being exact is the fact that it could have an impact in the case of a history rewrite, but I'm not clear enough on how this could or could not be important.